src/index.js

```diff
@@ -45,11 +53,11 @@ const brandColor = /** @type {const} */ ([174, 128, 255]);
 						{ value: 'ts', label: 'TypeScript' },
 					],
 				}),
-			useRouter: () =>
+			useRouter: ({ results }) => results.appType === 'spa' ?
 				prompts.confirm({
 					message: 'Use router?',
 					initialValue: false,
-				}),
+				}) : Promise.resolve(false),
```

Comment:
@rschristian do you know if there is a better way to do this? It looks like it goofs up our types

Reply:
I do not, no. [Clack does have some wonky types in places](https://github.com/natemoo-re/clack/issues/126) but it's been something I'm fine with `// @ts-ignore`'ing or whatever if the red squiggle is annoying for people.

We can ungroup the prompts, do something like [this](https://github.com/ascorbic/impala/blob/b2d1b5220527a66db98873ee38f17a73f9eea4de/packages/create-impala/src/cli.ts#L24-L35) to get better control flow, but it's perhaps a bit repetitive.